### ISSUE TITLE
Too bright

### ISSUE THREAD

**Issue by unobatbayar** [gree/unity-webview#691]
![Screenshot 2021-05-25 at 11 14 30](https://user-images.githubusercontent.com/38866249/119584609-d9038d00-be03-11eb-986c-6a0f475e0ade.png)

**Comment by unobatbayar** [comment]
Not sure why this is happening and can't seem to resolve.

**Comment by KojiNakamaru** [comment]
At least the sample app on the latest master works without any issue:

![image](https://user-images.githubusercontent.com/2645978/119595213-2984e580-be18-11eb-9522-d3eee896be15.png)

My environment is:
* macOS Big Sur (11.4)
* Unity 2020.3.8f1

Could you please provide a diff for the sample app that reproduces the issue?

**Comment by unobatbayar** [comment]
Yes, it works perfectly on a blank or the sample project. However, when I import to my project this occurs.

OK! Please wait, I'm investigating why this is happening.

**Comment by unobatbayar** [comment]
Environment the issue occurs in: 

- macOS Catalian(10.15.7)
- Unity 2019.4.11f1

Our project uses Universal Render Pipeline but removing this doesn't solve the issue..

**Comment by unobatbayar** [comment]
![Screenshot 2021-05-26 at 17 28 59](https://user-images.githubusercontent.com/38866249/119628239-e2164d80-be47-11eb-903c-c6fa9ec17ab8.png)

- [ ] Metal Editor Support
- [ ] Metal API Validation

Unchecking these resolved the brightness/contract issue. It's OK now! Thank you.
